### PR TITLE
MusicCast Presence

### PR DESCRIPTION
**Pull request by marinofranz**
Working Presence

### CODE REVIEW

**Comment by doomlerd** [comment]
delete the js file

File: websites/M/MusicCast/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "id": "598953747963707409"
+  },
+  "url": "musiccast.marinosite.xyz",
+  "description": {
+    "en": "Listen to the best tunes 24/7"
+  },
+  "service": "MusicCast",
+  "version": "1.0.0",
+  "logo": "https://cdn.marinosite.xyz/ps0t6c46.png",
+  "thumbnail": "https://cdn.marinosite.xyz/rxtizr28.png",
```
Comment (Alanexei) [inline websites/M/MusicCast/dist/metadata.json:13]: You will have to up the resolution of the thumbnail or make it less blurry.

Comment (doomlerd) [reply websites/M/MusicCast/dist/metadata.json:13]: it is intentional

Comment (marinofranz) [reply websites/M/MusicCast/dist/metadata.json:13]: It is intentional, however, I can remove the blur of you wish. I will be doing so.

**Pull request merged**